### ISSUE TITLE
Paragraph spacing is oddly big

### ISSUE THREAD

**Issue by johnwhitington** [ocaml/v2.ocaml.org#1472]
The paragraph spacing on ocaml.org is wider than the spacing between paragraphs and code, or paragraphs and images. This looks strange to me. Do others agree? See image below.

<img width="652" alt="spacing" src="https://user-images.githubusercontent.com/1702581/113881394-4dcb2b00-97b4-11eb-8121-13eb401dcd19.png">

**Comment by Srinithyee** [comment]
@johnwhitington it doesn't look disturbingly big to me. But, I could fix it if you'd like :)

**Comment by johnwhitington** [comment]
It's not that it's particularly big, it's that it's bigger than other spaces on the page. It groups the items on the page wrongly...

The problem could be fixed in several ways, but simply reducing the default paragraph space would do it.

**Comment by Srinithyee** [comment]
@johnwhitington Alright then!
@patricoferris This looks like a good first issue to me. I have already contributed to 3 simple issues. Should I leave this open for some other Outreachy applicant, still looking for one?

**Comment by gs0510** [comment]
@Srinithyee yes we can leave this for someone who hasn't contributed to a good-first-issue yet. Thanks!

**Comment by Srinithyee** [comment]
@SanduniK95 I see that you haven't been able to find any good-first-issue. Would you like to try this?

**Comment by gs0510** [comment]
@Deek9339 Do you want to take a look at this? Thanks! :)

**Comment by Lovelyfin00** [comment]
Hi
I'm an outreachy applicant and I would love to work on this.

**Comment by Deek9399** [comment]
> @Deek9339 Do you want to take a look at this? Thanks! :)

Hello @gs0510, I am currently working on [this issue](https://github.com/ocaml/ocaml.org/issues/1255). Hopefully, I will finish it by tomorrow.
Can I work on it after that?

**Comment by gs0510** [comment]
@Deek9399 Since you are already working on a good-first-issue, @Lovelyfin00 can work on it! :)

**Comment by Lovelyfin00** [comment]
Awesome
Thank you. I'll start now

**Comment by Lovelyfin00** [comment]
Hi
This is my first time actually working on open source. So, I just hope my question doesn't sound stupid.
How do I run make -j 8?
I tried typing it in the ocaml console on my pc nut it kept giving the error " No target specified and no make file found"
Thank you for your response

**Comment by gs0510** [comment]
@Lovelyfin00 you have to run `make -j 8` on your terminal, not inside the ocaml toplevel. When you do `ls` in your terminal, you should see a `Makefile` file in the output.
